06-setup-backup-restore/assets/manifests/triliovault-values.yaml

```diff
@@ -0,0 +1,77 @@
+## TrilioVault Operator
+registry: "eu.gcr.io/amazing-chalice-243510"
+
+operator-webhook-init:
+  repository: operator-webhook-init
+
+k8s-triliovault-operator:
+  repository: k8s-triliovault-operator
+
+tag: "2.6.7"
+
+tvkHelmRepo: ""
+
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+                - amd64
+
+image:
+  pullPolicy: Always
+tls:
+  secretName: "helm-client-certs"
+  verify: false
+  enable: false
+  keyFile: "tls.key"
+  certFile: "tls.crt"
+  caContent: ""
+  hostname: ""
+
+nameOverride: ""
+
+replicaCount: 1
+
+proxySettings:
+  PROXY_ENABLED: false
+  NO_PROXY: ""
+  HTTP_PROXY: ""
+  HTTPS_PROXY: ""
+  CA_BUNDLE_CONFIGMAP: ""
+
+podSpec:
+  hostIPC: false
+  hostNetwork: false
+  hostPID: false
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 1001
+
+securityContext:
+  allowPrivilegeEscalation: false
+  privileged: false
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+  runAsUser: 1001
+  capabilities:
+    drop:
+      - ALL
+
+installTVK:
+  enabled: true
+  applicationScope: Cluster
+  ingressConfig:
+    host: "tvk-doks.com"
+    tlsSecretName: ""
+    annotations: {}
+    ingressClass: ""
+  ComponentConfiguration:
+    ingressController:
+      enabled: true
+      service:
+        type: NodePort
+
```

Comment:
In the Helm values file, I would remove redundant sections not needed for this tutorial, like: `securityContext`, `podSpec`, `proxySettings`, etc. I would leave only the required ones for the scope of this tutorial. This way, the user won't get lost with other unnecessary details. You can always fetch the full Helm values file via: `helm show values triliovault-operator/k8s-triliovault-operator`.

So, in my opinion we should keep only the relevant and most important fields, like:

```yaml
## TrilioVault Operator
registry: "eu.gcr.io/amazing-chalice-243510"

operator-webhook-init:
  repository: operator-webhook-init

k8s-triliovault-operator:
  repository: k8s-triliovault-operator

tag: "2.6.7"

replicaCount: 1

installTVK:
  enabled: true
  applicationScope: Cluster
  ingressConfig:
    host: "tvk-doks.com"
    tlsSecretName: ""
    annotations: {}
    ingressClass: ""
  ComponentConfiguration:
    ingressController:
      enabled: true
      service:
        type: NodePort

```

Reply:
@v-ctiutiu Plan here is to keep the `triliovault-values.yaml` always updated since we have provided the same in the `helm install` command as well.
Do you think, we should ask users to pull the latest values file always, and then users should update the `triliovault-values.yaml` before running the `helm` command?
I have added some value in the note at line # 120.
Let me know if this looks good.